README.md

```diff
@@ -1,11 +1,11 @@
-# ppx_decco
+# Decco
 
 ## Latest version: v0.2
 **decco has been updated to support BS6!** If you need to use decco with BS5, install version ^0.1.0.
 
 ## What is it?
 
-Bucklescript PPX which generates JSON serializers and deserializers for user-defined types.
+A ReasonML / OCaml PPX which generates JSON serializers and deserializers for user-defined types.
```

Comment:
I think it's good to say BuckleScript instead of just Reason/OCaml since decco only supports BuckleScript and not native.

Reply:
Oh duh, you're totally right. I completely forgot that it uses Js.Json under the hood.